components/nifosg/particle.cpp

```diff
@@ -152,6 +152,7 @@ void ParticleColorAffector::operate(osgParticle::Particle* particle, double /* d
     osg::Vec4f color = mData.interpKey(time);
 
     particle->setColorRange(osgParticle::rangev4(color, color));
+    particle->setAlphaRange(osgParticle::rangef(color.w(), color.w()));
```

Comment:
This will just apply the transparency twice from my understanding of osgParticle. The fourth value of the color *is* used as transparency which is then multiplied by alpha range alpha.

Reply:
yes you're right